Tools/python/MetaSubmitter.py

```diff
@@ -207,18 +207,20 @@ def set_splitting(samples, dataset, jobtype='default', data_json=None, default_f
         # Shed/presel_splitting.py
         d = {'miniaod': {
                 'signal':           ( 1,     200),
-                'JetHT':            (15, 1350000),
```

Comment:
It looks like JetHT might have been accidentally dropped?

Reply:
I think I dropped it because I thought I won't use JetHT in the future...